homeassistant/data_entry_flow.py

```diff
@@ -408,7 +408,7 @@ def async_show_form(
         self,
         *,
         step_id: str,
-        data_schema: vol.Schema = None,
+        data_schema: vol.Schema | None = None,
```

Comment:
Is this handled in the frontend ?

Reply:
It's only a typing fix. Here's where the data_schema result key is used.

https://github.com/home-assistant/core/blob/5965b015dd0a5cb12dd0ed80a1abac764495cc36/homeassistant/data_entry_flow.py#L249-L250

https://github.com/home-assistant/core/blob/5965b015dd0a5cb12dd0ed80a1abac764495cc36/homeassistant/helpers/data_entry_flow.py#L42-L47

https://github.com/home-assistant/core/blob/5965b015dd0a5cb12dd0ed80a1abac764495cc36/homeassistant/components/auth/login_flow.py#L137-L140

https://github.com/home-assistant/core/blob/5965b015dd0a5cb12dd0ed80a1abac764495cc36/homeassistant/components/auth/mfa_setup_flow.py#L141-L144

Reply:
yeah, it was already handled, hence this change was minimal and merely typing.